<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des clients</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 217, 151);
font: 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Gestion des commandes </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_total">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statistiques">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stat par statut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajout">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ajout</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Users/skand/OneDrive/Bureau/design C++/Untitled-1.png</normaloff>../Users/skand/OneDrive/Bureau/design C++/Untitled-1.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>recherche par num</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>supp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_statut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recherche">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="total_de_la_commande">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total de la commande</string>
    </property>
   </widget>
   <widget class="QLabel" name="N_de_la_commande">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NÂ° de la commande</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_de_la_commande">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de la commande</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historique">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>historique</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modif">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFIER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tri">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>tri par total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherche_pushButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>100</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../ghaya/img.qrc">
      <normaloff>:/image/recherche.webp</normaloff>:/image/recherche.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="statut_de_la_commande">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Statut de la commande</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>507</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../ghaya/img.qrc">
      <normaloff>:/image/pdf.png</normaloff>:/image/pdf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-25</y>
      <width>1241</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ghaya/img.qrc">:/image/sk.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>liste</string>
     </attribute>
     <widget class="QTableView" name="tableView_r">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>751</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>chat</string>
     </attribute>
     <widget class="QLabel" name="label_chatbot_response">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chatbot">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="envoyer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>envoyer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ghaya/img.qrc">:/image/lo-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>formulaire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_menu">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../ghaya/img.qrc">
      <normaloff>:/image/dar.jpg</normaloff>:/image/dar.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>1052</x>
      <y>10</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../ghaya/img.qrc">
      <normaloff>:/image/quitter.png</normaloff>:/image/quitter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_total</zorder>
   <zorder>statistiques</zorder>
   <zorder>ajout</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_supp</zorder>
   <zorder>label_stat</zorder>
   <zorder>lineEdit_statut</zorder>
   <zorder>lineEdit_recherche</zorder>
   <zorder>total_de_la_commande</zorder>
   <zorder>N_de_la_commande</zorder>
   <zorder>date_de_la_commande</zorder>
   <zorder>historique</zorder>
   <zorder>lineEdit_N</zorder>
   <zorder>lineEdit_date</zorder>
   <zorder>modif</zorder>
   <zorder>tri</zorder>
   <zorder>recherche_pushButton</zorder>
   <zorder>statut_de_la_commande</zorder>
   <zorder>pdf</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_menu</zorder>
   <zorder>pushButton_log</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1285</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../ghaya/img.qrc"/>
 </resources>
 <connections/>
</ui>
